test/full/array-homogenous.ts

```diff
@@ -96,6 +96,14 @@ describe('array-homogenous', function() {
     testPackUnpackHomogeneousArray(-15.55, ONE, 2, true);
   });
 
+  it('floatExponentialOne', function() {
+    testPackUnpackHomogeneousArray(-1.552345411e+123, ONE, 2, true);
+  });
+
+  it('floatExponentialMany', function() {
+    testPackUnpackHomogeneousArray(-1.552345411e+123, MANY, 2, true);
+  });
+
```

Comment:
Another couple of test case would be worth adding is like `1e1` and `1e-1` for numbers that would generally not be expressed with this notation, but still could be.

Reply:
Makes sense. Added them in [test/full/number.ts](https://github.com/jgranstrom/zipson/pull/16/files#diff-355d3948d0d3c5fd6cd4870668dfe45d9218d77a141c2f78312c864c7b248d3a)